### PR TITLE
Sbt tests

### PR DESCRIPTION
**Pull request by SoulSilence**

### CODE REVIEW

File: src/sbt-it/build.sbt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+publishTo := {
+  val distributionRepository = "http://localhost:48080/storages/storage0/"
+  if (isSnapshot.value)
+    Some("snapshots" at distributionRepository + "snapshots")
+  else
+    Some("releases" at distributionRepository + "releases")
+}
+
+libraryDependencies ++= Seq(
```
Comment (carlspring) [inline src/sbt-it/build.sbt:24]: Could you add another dependency like `ch.qos.logback;logback-core:1.2.3` and add a check that the artifact has been resolved (check if the file is in the local cache of SBT... wherever that is)...?

**Comment by carlspring** [comment]
This pull looks great, thanks! I have a minor remark, other than that, it's ready for merging. I won't merge it before we have our Jenkins slaves set up to have SBT, though (it may be a few more days).

File: src/sbt-it/build.sbt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+version := "1.0-SNAPSHOT"
+
+publishMavenStyle := true
+
+credentials += Credentials("Strongbox Repository Manager",
+                           "localhost",
+                           "admin",
+                           "password")
+
+resolvers += "Strongbox" at "http://localhost:48080/storages/storage0/group-releases/"
```
Comment (carlspring) [inline src/sbt-it/build.sbt:14]: This should be `http://localhost:48080/storages/public/public-group`.

File: src/sbt-it/repositories [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[repositories]
+  local
+  releases: http://localhost:48080/storages/storage0/releases/
```
Comment (carlspring) [inline src/sbt-it/repositories:3]: This should be `http://localhost:48080/storages/public/public-group`.

**Comment by SoulSilence** [comment]
.

**Comment by carlspring** [comment]
Thanks! :)

**Pull request merged**